### PR TITLE
readme: Add "Fault Tolerance" section

### PR DESCRIPTION
**Pull request by nhoriguchi**
Based on #77, I have some update on `README.md` to describe fault tolerance. Current PR contains only about fault tolerance for backup's crash fault. The other cases like primary's fault and/or Byzantine fault are simply left as todo.

### CODE REVIEW

**Review by ynamiki — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -271,6 +272,60 @@ processes and release the occupied TCP ports:
...
+The above example shows a simple normal case of the consensus network.
+Our next interest is how the system behaves when some replicas are faulty.
+
+#### Crash Fault on Backup ####
+
+The simplest faulty case is crash fault on backup replicas. Note that
+we don't tolerate any type of fault on primary replica until view change
+operation is implemented.
+
+Let's restart the network, and keep in mind the process IDs of each
```
Comment (sergefdrv) [inline README.md:286]: Maybe `s/keep in mind/note`?

File: README.md [hunk 2]
```diff
@@ -271,6 +272,60 @@ processes and release the occupied TCP ports:
...
+$ bin/peer run 2 &
+[3] 16923
+```
+
+Make sure that all replicas are properly working by sending a request:
+
+```sh
+$ bin/peer request First request
+2019/04/10 02:46:06 Received Reply message from replica 1
+2019/04/10 02:46:06 Received Reply message from replica 0
```
Comment (sergefdrv) [inline README.md:303]: I think, for the sake of brevity, we'd better eliminate the lines of debug output.

**Review by sergefdrv — CHANGES_REQUESTED** [review]
@Naoya-Horiguchi Thanks for taking care of this. I left some comments.

File: README.md (hunk 1 above)
Comment (nhoriguchi) [reply README.md:286]: Applied, thank you.

File: README.md (hunk 2 above)
Comment (nhoriguchi) [reply README.md:303]: OK, removed these lines.

**Comment by nhoriguchi** [comment]
This pull request is revised by #85.

**Pull request closed without merging**